packages/pf3-component-mapper/src/form-fields/radio.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Radio } from 'patternfly-react';
+
+const RadioGroup = ({ FieldProvider, options, isDisabled, isReadOnly, input }) => options.map(option => (
+  <FieldProvider
+    key={ `${input.name}-${option.value}` }
+    name={ input.name }
+    value={ option.value }
+    type="radio"
+    render={ ({ input }) => (
+      <Radio { ...input } onChange={ () => { input.onChange(option.value); } } disabled={ isDisabled || isReadOnly }>{ option.label }</Radio>) }
+  />
+));
+
+RadioGroup.propTypes = {
+  FieldProvider: PropTypes.oneOfType([ PropTypes.node, PropTypes.func ]),
+  options: PropTypes.arrayOf(PropTypes.shape({
+    value: PropTypes.any,
+    label: PropTypes.node,
+  })),
+  isDisabled: PropTypes.bool,
+  isReadOnly: PropTypes.bool,
+  input: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+  }),
+};
+
+RadioGroup.defaultProps = {
+  options: [],
```

Comment:
Is this the line I am interested in?

Reply:
yes

Reply:
then what's the sugar around it? it seems HUGE for me :confused:

Reply:
You can assign default prop to anonymous function/component. 

Also if you notice all these three components in mappers are very similar. We plan to have "common" components that are shared between packages. This is just a step towards it.

Reply:
Okay, it's a little complicated for me, so I think I'm not the right person to review this :disappointed: